consensus/propagation/commitment.go

```diff
@@ -273,6 +282,12 @@ func (blockProp *Reactor) broadcastCompactBlock(cb *proptypes.CompactBlock, from
 		Message:   cb.ToProto(),
 	}
 
+	select {
```

Comment:
why is this moved here?

Reply:
it was moved here to also call it while we are the proposer